app/_locales/en/messages.json

```diff
@@ -1554,6 +1554,12 @@
   "developerOptions": {
     "message": "Developer Options"
   },
+  "developerOptionsEnableConfirmationsRedesignDescription": {
+    "message": "Enables or disables the confirmations redesign feature currently in development"
+  },
+  "developerOptionsEnableConfirmationsRedesignTitle": {
+    "message": "Confirmations Redesign"
+  },
```

Comment:
we can actually remove the locales here. Or I can follow-up to remove them to not block this PR. 

It was discussed that we do not need to support i18n in the Developer Options Settings since this is internal

ticket to remove translations:
https://github.com/MetaMask/metamask-extension/issues/24412